index.html

```diff
@@ -1,17 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID" crossorigin="anonymous"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){window.dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-HQREYRLMDN');
+    </script>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@400;700&display=swap" rel="stylesheet" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js" crossorigin="anonymous"></script>
     <title>Awesome Books</title>
 </head>
+
 <body>
+    <nav class="navContainer">
+        <h1 class="title1"><span>Awesome </span> Books</h1>
+        <img src="images/nav.svg" alt="navicon" class="navImg d-none">
+        <img src="images/closeNav.svg" alt="closeNav" class="closeImg d-none">
+        <ul class="navUl">
+            <li id="bookList" class="navLi pointer">Book list</li>
```

Comment:
- [ ] OPTIONAL
It is good practice to use **all lowercase with hyphens** separating words, for your CSS naming convertion